_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1105,34 +1273,43 @@ recommendations have not been implemented.
-1.16,<<iam_user_authentication>>,Use IAM groups and roles
-1.17,<<security_questions>>,Complete the contact details on the AWS account page
-1.18,<<security_questions>>,Complete the security contact information on the AWS account page
-1.19,<<iam_roles_for_instances>>,Use Gruntwork modules to ensure EC2 instances use roles for access
-1.20,<<iam_user_authentication>>,Use the `iam-groups` module to create a support group
-1.21,<<iam_user_authentication>>,Create IAM users with the `iam-users` module
-1.22,<<iam_user_authentication>>,Use the Gruntwork modules to create best-practices groups and roles
-2.1-2.4,<<cloudtrail>>,Use the Gruntwork CloudTrail wrapper module
-2.5,<<aws_config>>,Enable AWS Config for all regions
-2.6-2.7,<<cloudtrail>>,Use the Gruntwork CloudTrail wrapper module
-2.8,<<kms>>,Use the KMS module
-2.9,<<vpc_flow_logs>>,Use the VPC flow logs core module
-3.1-3.14,<<configure_monitoring>>,The CloudWatch Logs metrics filters wrapper module will satisfy each recommendation
-4.1,<<configure_networking>>,Use the Gruntwork VPC modules for a secure network configuration
-4.2,<<configure_networking>>,Use the Gruntwork VPC modules for a secure network configuration
-4.3,<<configure_networking>>,The cloud-nuke tool can remove all default security groups
-4.4,<<configure_networking>>,Gruntwork's VPC module creates least-privilege routing by default
+1.1,<<security_questions>>,Complete the contact details on the AWS account page
+1.2,<<security_questions>>,Complete the security contact information on the AWS account page
+1.3,<<security_questions>>,Answer the security questions on the AWS account page
+1.4,<<next_steps>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that no access key exists for the root user
+1.5,<<root_mfa>>,Manually configure MFA for the root user
+1.6,<<root_mfa>>,Use a Yubikey (or other hardware MFA) for the root user
+1.7,<<manual_steps>>,Take manual steps to complete this recommendation
+1.8-9,<<create_password_policy>>,Use the IAM password policy module
+1.10,<<configure_authentication>>,Configure authentication using SAML or IAM
+1.11,<<iam_user_authentication>>,Create IAM users with the `iam-users` module
+1.12,<<next_steps>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that there are no unused credentials
+1.13,TODO,Use SecurityHub
+1.14,<<next_steps>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that there are no unused access keys
+1.15,<<iam_user_authentication>>,Use IAM groups
+1.16,<<iam_user_authentication>>,Use the Gruntwork modules to create best-practices groups and roles
+1.17,<<iam_user_authentication>>,Use the `iam-groups` module to create a support group
+1.18,<<iam_roles_for_instances>>,Use Gruntwork modules to ensure EC2 instances use roles for access
+1.19,<<cleanup_expired_certs>>,Use Gruntwork modules to automatically remove expired certificates from IAM
+1.20,TODO(Ana),TODO
```

Comment:
@infraredgirl what did you want to say about the S3 stuff?

Reply:
Done - I've filled in the missing bits in the traceability matrix.